### ISSUE TITLE
[Feature Request] Defining & Accessing Relationship Attribute

### ISSUE THREAD

**Issue by opqopq** [neo4j-contrib/neomodel#20]
Hi All,

Great works. I do not know where to post my feature request, so it's gonna be an incident.

While checking the API, I was not able to find a way to both define & query the Property of a relationship.
Nevertheless, this is permitted on a Neo4J Level.
Any way one could implement it ? 
Or maybe subclassing RelationShipTo & From would suffice ? 

Thanks & congrats for this excellent work

**Comment by gchorn** [comment]
@robinedwards I like that `connect` supports defining properties on the relationship being created. However, I have noticed that `disconnect` does not support filtering by properties to narrow down which relationship should be deleted. 

As such, when I define multiple instances of a relationship type between two nodes, calling `disconnect` on that relationship type deletes _all_ relationships between the two nodes, whereas what I really want is to select and delete only one specific relationship. 

Is there any way to do this through the ORM that I've overlooked? I can't find it in the docs.
